packages/relay-test-utils/RelayTestSchema.js

```diff
@@ -15,7 +15,37 @@ const RelayTestSchemaPath = require('./RelayTestSchemaPath');
 const fs = require('fs');
 
 const {buildASTSchema, parse} = require('graphql');
+const {SchemaComposer} = require('graphql-compose');
 
-module.exports = buildASTSchema(
-  parse(fs.readFileSync(RelayTestSchemaPath, 'utf8'), {assumeValid: true}),
-);
+function buildSchema() {
+  // Compose upstream is going to add AST directives soon, making this simpler
+  const composer = new SchemaComposer();
```

Comment:
graphql-js supports extending schemas, can we just use that?

Reply:
Could you point me to that? extendSchema only support extending via AST.